### PR TITLE
Fix hover on orphan node

### LINKED ISSUE #51: hover is broken on an orphan node
hover is broken on an orphan node

as reported by @Tav0 in this PR https://github.com/visjs-community/visjs-network/pull/50/files

### PR DESCRIPTION
**Pull request by Tav0**
fix #51

### CODE REVIEW

**Comment by micahstubbs** [comment]
thanks for the contribution @Tav0!  will take a look at this

File: examples/events/interactionEvents/index.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ var nodes = new vis.DataSet([
   { id: 2, label: 'Node 2', title: 'I have a popup!' },
   { id: 3, label: 'Node 3', title: 'I have a popup!' },
   { id: 4, label: 'Node 4', title: 'I have a popup!' },
-  { id: 5, label: 'Node 5', title: 'I have a popup!' }
+  { id: 5, label: 'Node 5', title: 'I have a popup!' },
+  { id: 6, label: 'Node 6', title: 'I have a popup!' }
```
Comment (micahstubbs) [inline examples/events/interactionEvents/index.js]: @Tav0, so I tested this new example data on master (without the fix) and I am not able to reproduce the bug.  can you tell me more about when you see the bug (see hover broken)? 

Could you perhaps add another example to the PR that will show the bug if the fix is not present?

Comment (micahstubbs) [reply examples/events/interactionEvents/index.js]: some more details on what I tried in this issue https://github.com/visjs-community/visjs-network/issues/51

**Comment by Tav0** [comment]
Hi @micahstubbs sorry for the delay, here it is a gif of the issue. (master branch)
example-bug

**Comment by micahstubbs** [comment]
nice, thanks for adding the example!  I'll take a look at this, hopefully sometime this week.

**Comment by micahstubbs** [comment]
hi @Tav0, so checked out your branch

```bash
git checkout -b Tav0-fix/hover-orphan-node master
git pull https://github.com/Tav0/visjs-network.git fix/hover-orphan-node
```

and saw this:

50-orphan-node-hover--1-640px

1) the popup appears on the orphan node 6 added to the example (this works on master too)
2) the popup appears on the connected node 5 (this worked on master too)
3) the popup does not appear on the newly added `new` node (this is broken on master too)

**Comment by micahstubbs** [comment]
@Tav0 could you take another look, and see if you can produce an example where 3) works too?

**Comment by micahstubbs** [comment]
it looks to me like when we mouse over the `new` node, the `hidePopup` event is fired too soon.  It fires before we mouse out from the new node.
50-hidepopup-event-fired-too-soon

**Comment by micahstubbs** [comment]
contrast this with the behavior of `Node 6`, where the `hidePopup` event is not fired until we mouse out.  This is the expected behavior.  this and all previous examples are taken from a local server running the code in this PR.

50-node-6-no-hidepopup-event-until-mouse-out

**Comment by micahstubbs** [comment]
@Tav0 let's keep at this - I think we are close to a solution here 😄 📈

**Comment by micahstubbs** [comment]
will close this PR, since I was not able to reproduce the fix.

@Tav0 please do re-open this PR if you would like to help us verify the fix 😄

**Pull request closed without merging**